From 01b93b76d0bbb039f9f9d09a94bc1e26f53d7ffb Mon Sep 17 00:00:00 2001
From: gasinvein <gasinvein@gmail.com>
Date: Mon, 21 Jun 2021 00:26:50 +0300
Subject: [PATCH] Update metainfo

Put Tuned service in a separate paragraph and mention that it must be installed.

Add bugtracker URL.
---
 assets/metainfo/tuned-switcher.metainfo.xml.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/assets/metainfo/tuned-switcher.metainfo.xml.in b/assets/metainfo/tuned-switcher.metainfo.xml.in
index ec4db05..2d5cbae 100644
--- a/assets/metainfo/tuned-switcher.metainfo.xml.in
+++ b/assets/metainfo/tuned-switcher.metainfo.xml.in
@@ -12,7 +12,10 @@ SPDX-License-Identifier: CC0-1.0
     <description>
         <p>
             Tuned Switcher is a simple utility to manipulate the Tuned service.
+        </p>
+        <p>
             Tuned is daemon for monitoring and adaptive tuning of system devices.
+            In order to use this program, the daemon must be installed on your system.
         </p>
     </description>
     <screenshots>
@@ -21,6 +24,7 @@ SPDX-License-Identifier: CC0-1.0
         </screenshot>
     </screenshots>
     <url type="homepage">@CMAKE_PROJECT_HOMEPAGE_URL@</url>
+    <url type="bugtracker">@CMAKE_PROJECT_HOMEPAGE_URL@/issues</url>
     <update_contact>@CMAKE_PROJECT_HOMEPAGE_URL@</update_contact>
     <launchable type="desktop-id">@APPLET_LAUNCHER@</launchable>
     <content_rating type="oars-1.0" />
-- 
2.31.1

